aries-site/src/examples/components/pagination/PaginationTableExample.js

```diff
@@ -487,13 +487,15 @@ const columns = [
       </Box>
     ),
     render: datum => (
-      <Box pad={{ vertical: 'xsmall' }}>
+      <Box direction="row" gap="xxsmall" pad={{ vertical: 'xsmall' }}>
```

Comment:
For some reason it looks like the rows are taller now than before but I'm not sure why on first glance. Don't think the row height should increase:

BEFORE:
<img width="1228" alt="Screen Shot 2021-05-06 at 4 12 37 PM" src="https://user-images.githubusercontent.com/12522275/117376399-e6250000-ae85-11eb-9335-d7eeafe48ba2.png">

AFTER:
<img width="1218" alt="Screen Shot 2021-05-06 at 4 12 28 PM" src="https://user-images.githubusercontent.com/12522275/117376409-ea511d80-ae85-11eb-93eb-a20c8572c548.png">

Reply:
oh good catch I didnt even see that let me go back and inspect

Reply:
so it was the `box` around `meter`  had a vertical `pad` so that was also going to the `text` so I closed that `box` before `text`